package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zowe/imperative",
-  "version": "5.7.5",
+  "version": "5.8.0",
```

Comment:
Please revert the version bump, our CI will bump the version automatically when the PR is merged.

Reply:
Refreshed package.json from the imperative master branch